<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>219</y>
      <width>436</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SplitHCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
background: transparent;
height: 20px;
border-radius: 0px;
}




QSlider::handle:horizontal {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,stop:0 #ffe000,stop:1 #799f0c);
border: 2px solid #000;
width: 15px;
border-radius: 8px;
}


</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="length">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>213</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
border: 2px solid #eeeeee;
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>225</y>
      <width>421</width>
      <height>8</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0#ffffff, stop:1 #abbaab);
border: 1px solid #000;
}
QProgressBar::chunk 
{
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,stop:0 #7f00ff,stop:1 #e100ff);

border: 1px solid #000;

}    </string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>766</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#272822;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>436</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #ffffff;
padding-left: 10;
background-color: rgb(118, 118, 118);
background:#272822;







</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Your Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BackgroundBehind_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>736</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0#ffffff, stop:1 #abbaab);
border-radius:5px;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoPredictiveText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #00FFFFFF ;
</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="help_btn">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#272822;
color: rgb(255, 255, 255);
border-radius:8px;</string>
       </property>
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMinimize">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#272822;
border-radius:8px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMaximiz">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#272822;

border-radius:8px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#272822;
border-radius:8px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>150</y>
      <width>556</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QProgressBar
{
border-radius:8px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0#ffffff, stop:1 #abbaab);

}
QProgressBar::chunk 
{
border-radius:8px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0#FDFC47, stop:1 #24FE41);

}    </string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="refresh">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>105</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right: 2px solid #eeeeee;
border-left: 2px solid #eeeeee;
border-top: 2px solid #eeeeee;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="refresh_2">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>105</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonrefresh">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>105</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>PVS/refresh.png</normaloff>PVS/refresh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtoncopy">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>105</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00FFFFFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>PVS/copy.png</normaloff>PVS/copy.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>113</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5;

</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="symbols_2">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="uppercase">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Uppercase</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lowercase">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Lowercase</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="numbers">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Numbers</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="symbols">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Symbols</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>uppercase</zorder>
    <zorder>lowercase</zorder>
    <zorder>numbers</zorder>
    <zorder>symbols</zorder>
    <zorder>symbols_2</zorder>
   </widget>
   <widget class="QLabel" name="bordertick">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>105</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #eeeeee;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bordertick_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>556</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #eeeeee;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="refresh_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>105</y>
      <width>556</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #eeeeee;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #00FFFFFF ;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>refresh_3</zorder>
   <zorder>bordertick</zorder>
   <zorder>bordertick_2</zorder>
   <zorder>BackgroundBehind_2</zorder>
   <zorder>progressBar</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>lineEdit</zorder>
   <zorder>frame</zorder>
   <zorder>refresh</zorder>
   <zorder>refresh_2</zorder>
   <zorder>pushButtonrefresh</zorder>
   <zorder>pushButtoncopy</zorder>
   <zorder>length</zorder>
   <zorder>progressBar_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>verticalWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>length</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>progressBar_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>symbols_2</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMinimize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
